<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientExplorerWidget</class>
 <widget class="QWidget" name="PatientExplorerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Patient explorer</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>561</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="editPath">
        <property name="toolTip">
         <string/>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Enter path to DICOM files...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btnBrowse">
      <property name="text">
       <string>&amp;Browse...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTreeWidget" name="treePatients">
      <column>
       <property name="text">
        <string>Patients</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
